### PR TITLE
Cayenne -> datil-dev, changed pages + comments

### PR DESCRIPTION
**Pull request by awisniew207**
Replaced instances of:
litNetwork: LitNetwork.Cayenne -> litNetwork: "datil-dev"
litNetwork: 'cayenne' -> litNetwork: "datil-dev"
litNetwork: "cayenne" -> litNetwork: "datil-dev"

Mentions of Cayenne that were not changed. Please let me know if you think they should be:
https://developer.litprotocol.com/integrations/aa/alchemy-account-kit
Feature matrices (being compared to jalapeno and serrano)
Migration guides (pre-v4)
Networks page (inaccessible from search/sidebar) (https://developer.litprotocol.com/network/state-of-network)

^ Essentially mentions of features specific to Cayenne.

Effected docs:
https://developer.litprotocol.com/sdk/access-control/quick-start
https://developer.litprotocol.com/sdk/access-control/encryption <- Not accessible from search/sidebar
https://developer.litprotocol.com/sdk/authentication/auth-sig <- Auth Sig depricated, but changed regardless
https://developer.litprotocol.com/sdk/authentication/session-sigs/get-session-sigs <- should getPkpSessionSigs be added?
https://developer.litprotocol.com/sdk/serverless-signing/broadcast-and-collect
https://developer.litprotocol.com/sdk/serverless-signing/combining-decryption-shares
https://developer.litprotocol.com/sdk/serverless-signing/combining-signatures 
https://developer.litprotocol.com/sdk/serverless-signing/conditional-signing
https://developer.litprotocol.com/sdk/serverless-signing/encryption
https://developer.litprotocol.com/sdk/serverless-signing/fetch
https://developer.litprotocol.com/sdk/serverless-signing/get-rpc-url <- Only available on Cayenne -> datil-dev
https://developer.litprotocol.com/sdk/serverless-signing/processing-validation
https://developer.litprotocol.com/sdk/serverless-signing/run-once
https://developer.litprotocol.com/sdk/wallets/auth-methods/lit-auth-methods/add-remove-auth-methods <- "SessionSig (deprecated)" ???
https://developer.litprotocol.com/sdk/wallets/auth-methods/lit-auth-methods/email-sms
https://developer.litprotocol.com/sdk/wallets/claimable-keys/usage
https://developer.litprotocol.com/sdk/wallets/walletconnect
https://developer.litprotocol.com/sdk/wrapped-keys/exporting-wrapped-key
https://developer.litprotocol.com/sdk/wrapped-keys/generating-wrapped-key
https://developer.litprotocol.com/sdk/wrapped-keys/getting-wrapped-key-metadata
https://developer.litprotocol.com/sdk/wrapped-keys/importing-key
https://developer.litprotocol.com/sdk/wrapped-keys/sign-message
https://developer.litprotocol.com/sdk/wrapped-keys/sign-transaction

If you would like for me to add any additional changes please let me know!

### CODE REVIEW

**Comment by spacesailor24** [comment]
- https://developer.litprotocol.com/integrations/aa/alchemy-account-kit should probably be updated, but I'm not even sure if this code still works
- Feature matrices (being compared to jalapeno and serrano)
    - If you're talking about [this page](https://developer.litprotocol.com/network/networks/testnet), then we should probably add `datil-dev` to it
- Migration guides (pre-v4)
    - I think we should update this as we don't want anyone using cayenne anymore @a1ttech thoughts?
- Networks page (inaccessible from search/sidebar) (https://developer.litprotocol.com/network/state-of-network)
    - This page is outdated and can probably be removed since we have [this Networks page](https://developer.litprotocol.com/category/networks)

**Comment by spacesailor24** [comment]
> https://developer.litprotocol.com/sdk/access-control/encryption <- Not accessible from search/sidebar

[This page](https://developer.litprotocol.com/sdk/access-control/quick-start) covers the same topic, I think we can remove the above page from the repo @a1ttech for confirmation

**Comment by spacesailor24** [comment]
> https://developer.litprotocol.com/sdk/authentication/auth-sig <- Auth Sig depricated, but changed regardless

A while back we did a search throughout the docs for `auth sig` and added the disclaimer that they were deprecated for authorization to use the Lit network. However, this was a rushed job and needs to be redone, because Auth Sigs are still able to be used for authentication and how to produce them and when to use them needs to be documented

We've had this task to revisit all the mentions of Auth Sigs in the docs on the todo list for a while, there's just been a lot of higher priority stuff recently

@a1ttech @alexvotofuture this might be a good task for Andrew to work on

**Comment by spacesailor24** [comment]
> https://developer.litprotocol.com/sdk/authentication/session-sigs/get-session-sigs <- should getPkpSessionSigs be added?

Yes, we need to document `getPkpSessionSigs` along with any other new methods that have been added to the SDK in v6

**Comment by spacesailor24** [comment]
> https://developer.litprotocol.com/sdk/serverless-signing/get-rpc-url <- Only available on Cayenne -> datil-dev

I think this is actually supported on all networks now. @DashKash54 or @Ansonhkg to confirm

**Comment by spacesailor24** [comment]
> https://developer.litprotocol.com/sdk/wallets/auth-methods/lit-auth-methods/add-remove-auth-methods <- "SessionSig (deprecated)" ???

Yea I don't understand this either. We should check with the SDK team for context on this and update the doc to be more specific

File: docs/sdk/wrapped-keys/exporting-wrapped-key.md [hunk 1]
```diff
@@ -156,14 +156,14 @@ const ethersSigner = new ethers.Wallet(
 
 ### Instantiating a `LitNodeClient`
 
-Here we are instantiating an instance of `LitNodeClient` and connecting it to the `cayenne` Lit network.
+Here we are instantiating an instance of `LitNodeClient` and connecting it to the `cdatil-dev` Lit network.
```
Comment (spacesailor24) [inline docs/sdk/wrapped-keys/exporting-wrapped-key.md:159]:
```suggestion
Here we are instantiating an instance of `LitNodeClient` and connecting it to the `datil-dev` Lit network.
```

**Review by spacesailor24 — CHANGES_REQUESTED** [review]
Anywhere we're installing packages specifically to be used on `datil-dev` we need to add the NPM tag `datil-dev` to the install command. For example,

this

```ts
yarn install @lit-protocol/lit-node-client
```

needs to become

```ts
yarn install @lit-protocol/lit-node-client@datil-dev
```

---

Once Datil becomes the default for the SDK this won't be necessary, but I'm not sure when that is

**Comment by awisniew207** [comment]
Thanks for the feedback @spacesailor24, some notes for the recent push on this PR:

Done:
Alchemy updated
Removed (deprecated) from SessionSig
Removed mentions of cayenne from previous migration guides
Updated testnet feature matrix :
https://developer.litprotocol.com/network/networks/testnet
Please check info on it. Not sure if we want to add/change anything.

@lit-protocol/lit-node-client -> @lit-protocol/lit-node-client@datil-dev
^ (only 1 instance)

TBD:
New methods introduced in V6
Revisit mentions of Auth Sigs in docs
This is an old migration feature matrix, fine to leave the same?:
https://developer.litprotocol.com/network/feature-matrix
Confirmation on serverless signing support for networks other than cayenne

TBD Page removals:
https://developer.litprotocol.com/sdk/access-control/encryption
https://developer.litprotocol.com/network/state-of-network

**Comment by awisniew207** [comment]
Changed the pressing implementations for this PR. Cayenne should be only mentioned in some old matrices and that it is now deprecated.

File: docs/integrations/aa/alchemy-account-kit.md [hunk 2]
```diff
@@ -66,15 +66,15 @@ values={[
 <TabItem value="npm">
 
 ```js
-npm i @lit-protocol/pkp-ethers@cayenne
+npm i @lit-protocol/pkp-ethers@datil-dev
```
Comment (spacesailor24) [inline docs/integrations/aa/alchemy-account-kit.md:69]:
```suggestion
npm i @lit-protocol/pkp-ethers
```

Version `6.1.0` of the SDK (that supports Datil-dev) is not installed from NPM by default

File: docs/integrations/aa/alchemy-account-kit.md [hunk 3]
```diff
@@ -66,15 +66,15 @@ values={[
...
 ```
 
 </TabItem>
 
 <TabItem value="yarn">
 
 ```js
-yarn add @lit-protocol/pkp-ethers@cayenne
+yarn add @lit-protocol/pkp-ethers@datil-dev
```
Comment (spacesailor24) [inline docs/integrations/aa/alchemy-account-kit.md:77]:
```suggestion
yarn add @lit-protocol/pkp-ethers
```

File: docs/integrations/aa/alchemy-account-kit.md [hunk 4]
```diff
@@ -92,19 +92,19 @@ values={[
 <TabItem value="npm">
 
 ```js
-npm i @lit-protocol/lit-node-client@cayenne
-npm i @lit-protocol/crypto@cayenne
-npm i @lit-protocol/auth-helpers@cayenne
+npm i @lit-protocol/lit-node-client@datil-dev
+npm i @lit-protocol/crypto@datil-dev
+npm i @lit-protocol/auth-helpers@datil-dev
```
Comment (spacesailor24) [inline docs/integrations/aa/alchemy-account-kit.md:97]:
```suggestion
npm i @lit-protocol/lit-node-client
npm i @lit-protocol/crypto
npm i @lit-protocol/auth-helpers
```

File: docs/integrations/aa/alchemy-account-kit.md [hunk 5]
```diff
@@ -92,19 +92,19 @@ values={[
...
 
 <TabItem value="yarn">
 
 ```js
-yarn add @lit-protocol/lit-node-client@cayenne
-yarn add @lit-protocol/crypto@cayenne
-yarn add @lit-protocol/auth-helpers@cayenne
+yarn add @lit-protocol/lit-node-client@datil-dev
+yarn add @lit-protocol/crypto@datil-dev
+yarn add @lit-protocol/auth-helpers@datil-dev
```
Comment (spacesailor24) [inline docs/integrations/aa/alchemy-account-kit.md:107]:
```suggestion
yarn add @lit-protocol/lit-node-client
yarn add @lit-protocol/crypto
yarn add @lit-protocol/auth-helpers
```

File: docs/network/networks/testnet.md [hunk 6]
```diff
@@ -19,8 +19,9 @@ You can set your Lit Network in the [LitNodeClient config of the Lit SDK](../../
...
+| Cayenne | Centralized test network. Keys are not persistent and will be deleted.  This network does not enforce payment and can be used for free, for testing. | BLS, ECDSA | Encryption, User Wallets (PKPs), Serverless Signing (Lit Actions) | Not Live | V4+ | Deprecated. Do not build new apps that use this network.| [cayenne](https://github.com/LIT-Protocol/networks/tree/main/cayenne) |
+| Manzano | Decentralized test network. No persistency guarantees.  Mirrors Habanero code and configuration.  Payment is enforced. | BLS, ECDSA | Encryption, User Wallets (PKPs), Serverless Signing (Lit Actions) | Live | V4+ | Good to use | [manzano](https://github.com/LIT-Protocol/networks/tree/main/manzano) 
+| DatilDev | Centralized test network. Keys are not persistent and will be deleted. This network does not enforce payment and can be used for free, for testing. | BLS, ECDSA | Encryption, User Wallets (PKPs), Serverless Signing (Lit Actions) | Live | V6+ | Good to use | [datil-dev](https://github.com/LIT-Protocol/networks/tree/main/datil-dev) |
```
Comment (spacesailor24) [inline docs/network/networks/testnet.md:24]:
```suggestion
| Datil-dev | Centralized test network. Keys are not persistent and will be deleted. This network does not enforce payment and can be used for free, for testing. | BLS, ECDSA | Encryption, User Wallets (PKPs), Serverless Signing (Lit Actions) | Live | V6+ | Good to use | [datil-dev](https://github.com/LIT-Protocol/networks/tree/main/datil-dev) |
```

File: docs/sdk/access-control/encryption.md
Comment (spacesailor24) [inline docs/sdk/access-control/encryption.md:1]: Why was this page removed?

File: docs/sdk/serverless-signing/conditional-signing.md [hunk 7]
```diff
@@ -21,10 +21,10 @@ The below example will check if the user has at least 1 Wei on Ethereum, only re
 The `ethers.utils.arrayify(ethers.utils.keccak256(...)` can be used to convert the `toSign` data to the correct format.
 :::
 
-Installed the latest client on `cayenne`
+Installed the latest client on `datil-dev`
 
 ```bash
-yarn install @lit-protocol/lit-node-client
+yarn install @lit-protocol/lit-node-client@datil-dev
```
Comment (spacesailor24) [inline docs/sdk/serverless-signing/conditional-signing.md:27]:
```suggestion
yarn install @lit-protocol/lit-node-client
```

File: docs/sdk/serverless-signing/encryption.md [hunk 8]
```diff
@@ -3,7 +3,7 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
 # Decrypting and Combining Within an Action
 
 :::info
-    Only available on the `cayenne` network
+    Only available on the `datil-dev` network
```
Comment (spacesailor24) [inline docs/sdk/serverless-signing/encryption.md:6]:
```suggestion
    Only available on the Lit networks:
        - `habanero`
        - `manzano`
        - `datil-dev` 
```

**Review by spacesailor24 — CHANGES_REQUESTED** [review]

File: docs/sdk/access-control/encryption.md
Comment (awisniew207) [reply docs/sdk/access-control/encryption.md:1]: You mentioned another page (https://developer.litprotocol.com/sdk/access-control/quick-start) covers the same topics
https://github.com/LIT-Protocol/docs/pull/307#issuecomment-2207532222

**Review by spacesailor24 — APPROVED** [review]

**Pull request merged**